<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainPreview</class>
 <widget class="QMainWindow" name="MainPreview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>741</width>
      <height>481</height>
     </rect>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>701</width>
       <height>461</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="videoPanel_1" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="ipListWidget">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>600</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>192.168.1.65</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>192.168.7.12</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="viewerLayout" native="true">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="3">
      <widget class="QLineEdit" name="portLineEdit"/>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ipLineEdit"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>port</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
